explainer-beforematch.md

```diff
@@ -191,6 +191,88 @@ In general, we believe that the `beforematch` event does not pose a privacy prob
 However, the privacy aspect of the event should be discussed in more detail via
 formal security reviews.
 
+### Responses to DOM and style changes in the `beforematch` event handler
+
+The scrolling behavior exhibited by find-in-page, ScrollToTextFragment, and
+ElementFragment after style and DOM changes are made by `beforematch` event
+handlers is not well specified yet and is being discussed here: 
+https://github.com/WICG/display-locking/issues/150
+
+Here is a list of `beforematch` handler use cases that could affect scrolling:
+1. **Removed from DOM**: The `beforematch` event handler removes the
+target element from the DOM.
+2. **Reparented in DOM**: The `beforematch` event handler removes the target
+element from the DOM, and re-adds it further down the tree such that the
+location to scroll to is different.
+3. **display: none**: The `beforematch` event handler adds the style
+`display: none` to the target element.
+4. **visibility: hidden**: The `beforematch` event handler adds the style
+`visibility: hidden` to the target element.
+5. **subtree-visibility: hidden-matchable -> visible**: The target element has
+the style `subtree-visibility: hidden-matchable` before the `beforematch` event
+is fired on the target element, and the `beforematch` event handler changes the
+style value from `hidden-matchable` to `visible`.
+
+
+Here is the current scrolling behavior in Chromium for each of those cases in
```

Comment:
I'd add a small disclaimer here, something like

This is subject to change as we evolve and improve the implementation.

Reply:
done.